docs/site/soap-calculator-tutorial-add-controller.md

```diff
@@ -3,12 +3,12 @@ lang: en
 title: 'Add a Controller'
 keywords: LoopBack 4.0, LoopBack 4
 sidebar: lb4_sidebar
-permalink: /doc/en/lb4/soap-calculator-add-controller.html
+permalink: /doc/en/lb4/soap-calculator-tutorial-add-controller.html
 ---
 
 ### Add a Controller
 
-![soap-calculator-add-service](../../docs/img/loopback-example-soap-calculator_figure4.png)
+![soap-calculator-add-service](../img/loopback-example-soap-calculator_figure4.png)
```

Comment:
The images right now live at `loopback-next/docs/img` ... this folder is not copied into `loopback.io` during the jekyll build process. The images in the above directory need to be moved to `/loopback-next/docs/site/imgs/`. 

You can test this locally by doing the following from `loopback-next` top level. 
```
npm run build:site
cd sandbox/loopback.io
npm start
// Visit http://127.0.0.1:4001/doc/en/lb4/soap-calculator-tutorial.html#overview
```

Then this should be good to land with the links fixed. Thanks @marioestradarosa !

Reply:
Thanks for the explanation. 👍